lib/ansible/modules/windows/win_share.ps1

```diff
@@ -47,6 +47,7 @@ $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "b
 
 $name = Get-AnsibleParam -obj $params -name "name" -type "str" -failifempty $true
 $state = Get-AnsibleParam -obj $params -name "state" -type "str" -default "present" -validateset "present","absent"
+$appendRules = Get-AnsibleParam -obj $params -name "appendRules" -type "bool" -default $false
```

Comment:
We seem to have settled on using `action` as an option name for similar modules that need to add, remove, or set from a list. I would suggest we use `rule_action` with those 3 options and default to `set` to ensure backwards compatibility. You don't need to implement the `remove` if you don't want to but having `set` and `add` should satisfy the feature request.

Reply:
done